packages/react-broadcast/src/components/test-context-provider/test-context-provider.tsx

```diff
@@ -0,0 +1,52 @@
+import type {
+  TFlags,
+  TReconfigureAdapter,
+  TAdapterStatus,
+  TAdapterIdentifiers,
+} from '@flopflip/types';
+
+import React from 'react';
+import { AdapterConfigurationStatus, AdapterSubscriptionStatus } from '@flopflip/types';
+import { createAdapterContext, AdapterContext } from '@flopflip/react';
+import { createIntialFlagsContext, FlagsContext } from '../flags-context';
+
+type TProps = {
+  children: React.ReactNode;
+  flags: TFlags;
+  adapterIdentifiers?: TAdapterIdentifiers & string[];
+  reconfigure?: TReconfigureAdapter;
+  status?: TAdapterStatus;
+};
+
+const defaultProps: Pick<TProps, 'recondigure' | 'status'> = {
+  adapterIdentifiers: ['test'],
+  status: {
+    subscriptionStatus: AdapterSubscriptionStatus.Subscribed,
+    configurationStatus: AdapterConfigurationStatus.Configured,
+  }
+};
+
+const TestContextProvider = (props: TProps) => {
```

Comment:
Can you clarify what makes this setup specific to testing?

Reply:
Yes. I can't use it internally. The two contexts (flags and adapter) are internally in different parts of the tree. The adpater context higher up in the react package (cause broadcast and redux use it) and the flags context within the respective package.